src/control-plane/backend/click-stream-api.ts

```diff
@@ -406,6 +412,15 @@ export class ClickStreamApiConstruct extends Construct {
               arnFormat: ArnFormat.COLON_RESOURCE_NAME,
             }, Stack.of(this),
           ),
+          Arn.format(
+            {
+              resource: 'stateMachine',
+              service: 'states',
+              region: '*',
+              resourceName: `${CLICKSTREAM_SEGMENTS_WORKFLOW_PREFIX}*`,
```

Comment:
You can merge this resource into the above policies.

Reply:
I'm appending new resource to existing resources list. Is there other place I can merge it to?